tradingpost.py

```diff
@@ -106,7 +109,13 @@ def price(self, msg, args):
     def pwp(self, msg, args):
         '''Fetches the PWP score and bye eligibility for a DCI number :trophy:'''
```

Comment:
I guess we should change this to ~`Fetches the current PlaneswalkerPoints for a DCI number`?